std/testing/runner.ts

```diff
@@ -116,6 +116,28 @@ export interface RunTestModulesOptions extends RunTestsOptions {
   allowNone?: boolean;
 }
 
+/**
+ * Renders test file that will be run.
+ *
+ * It's done to optimize compilation of test files, because
+ * dynamically importing them one by one takes very long time.
+ * @TODO(bartlomieju): try to optimize compilation by reusing same compiler host
+ * multiple times
+ * @param testModules
+ */
+function renderTestFile(testModules: string[]): string {
+  let testFile = "";
+
+  for (const testModule of testModules) {
+    // NOTE: this is intentional that template string is not used
+    // because of TS compiler quirkness of trying to import it
+    // rather than treating it like a variable
+    testFile += 'import "' + testModule + '"\n';
+  }
+
+  return testFile;
+}
```

Comment:
This is a hack but let's just go with it for now. It's much better to have the std tests running in "cargo test"